### PR TITLE
[kops] add autoscaler support

### PR DESCRIPTION
**Pull request by alebabai**
## what

- add additional cluster policy
- add cloudLabels

## why
#242

### CODE REVIEW

**Comment by alebabai** [comment]
@osterman if necessary to have cluster autoscaler optional?

File: rootfs/templates/kops/default.yaml [hunk 1]
```diff
@@ -32,6 +45,10 @@ spec:
   channel: stable
   cloudLabels:
     Cluster: {{ getenv "KOPS_CLUSTER_NAME" }}
+    {{- if bool (getenv "KOPS_CLUSTER_AUTOSCALER_ENABLED" "false") }}
+    k8s.io/cluster-autoscaler/enabled: "true"
+    kubernetes.io/cluster/{{ getenv "KOPS_CLUSTER_NAME" }}: "owned"
```
[the remark this answers is not in the document]
Comment (alebabai) [reply rootfs/templates/kops/default.yaml]: These labels are needed to support auto-discovery feature.
https://github.com/helm/charts/tree/master/stable/cluster-autoscaler#auto-discovery

[the remark this answers is not in the document]
Comment (osterman) [reply rootfs/templates/kops/default.yaml]: Aha, so this is something specific that that chart implementation for the cluster-autoscaler.

File: rootfs/templates/kops/default.yaml [hunk 2]
```diff
@@ -13,6 +13,19 @@ spec:
             ],
             "Effect": "Allow",
             "Resource": ["*"]
+          {{- if bool (getenv "KOPS_CLUSTER_AUTOSCALER_ENABLED" "false") }}
```
[the remark this answers is not in the document]
Comment (aknysh) [reply rootfs/templates/kops/default.yaml]: ah sorry, the way it's done works too :)
just not easy to follow

**Pull request merged**